pkg/util/ssh.go

```diff
@@ -199,10 +200,15 @@ func runSSHCommand(dialer sshDialer, cmd, user, host string, signer ssh.Signer)
 		} else {
 			// Some other kind of error happened (e.g. an IOError); consider the
 			// SSH unsuccessful.
-			err = fmt.Errorf("failed running `%s` on %s@%s: '%v'", cmd, user, host, err)
+			err = fmt.Errorf("failed running `%s` on %s: '%v'", cmd, remote, err)
 		}
 	}
-	return bout.String(), berr.String(), code, err
+	stdout, stderr := bout.String(), berr.String()
+	glog.Infof("[%s] stdout:    %q", remote, stdout)
+	glog.Infof("[%s] stderr:    %q", remote, stderr)
+	glog.Infof("[%s] exit code: %d", remote, code)
+	glog.Infof("[%s] error:     %v", remote, err)
```

Comment:
Oh, sorry, I assumed this stuff only got printed on an error.

If this happens even in the success case, then this should be V(8) or something really high.

We display V(2) and lower by default.

I would recommend packaging this stuff in the error / displaying it only if there was an error. Otherwise this is going to cause a lot of log spam.